StudentAdvisementSystem/src/java/edu/uco/teamfreelabor/UserBean.java

```diff
@@ -190,9 +187,7 @@ public String insert() throws SQLException
             ps.setString(5, lastName);
             ps.setString(6, id);
             ps.setString(7, major);
-            //advisementStatus = "Done";
-            //we dont have advisement status on register.xhtml right now so I am using a string literal
-            ps.setString(8, advisementStatus);
+            ps.setString(8, advisementStatus); //Default will set the default value in sql -- How to use fem java ?

```

Comment:
I can't use the default column value from java but it works in mysql. Just changed the starting value to be the same until I can find a fix